about/package-scope.md

```diff
@@ -30,9 +28,229 @@ open a [pre-submission inquiry using a GitHub Issue](https://github.com/pyOpenSc
+
+We will update this page as our review process evolves.
+
+```{note}
+Many of the example packages below perform tasks that might fit in multiple
+categories. There are just there to provide you with a flavor of the types
+of packages that would fall into that category.
+```
+
+### Data retrieval
+Packages for accessing and downloading data from online sources. This category
+includes wrappers for accessing APIs.
+
+Our definition of scientific applications is broad, including data storage
+services, journals, and other remote servers, as many data sources may be of
+interest to scientists. However, retrieval packages should be focused on data
+sources / topics, rather than services.
+
+*For example a general client for Amazon Web Services data storage would not be in-scope.*
+
+* Examples: [OpenOmics](https://github.com/pyOpenSci/software-submission/issues/31)
+
+
+### Data extraction
+
+These packages aid in retrieving data from unstructured sources such as text,
+images, and PDFs. They might also parse scientific data types and outputs from
+scientific equipment.
+
+* Examples: [devicely](https://github.com/pyOpenSci/software-submission/issues/37), [jointly](https://github.com/pyOpenSci/software-submission/issues/45)
+
+### Data processing & munging
+
+Data munging tools are tools that support processing data discussed above. This
+category focuses on tools for handling data in specific  formats that scientists
+may be interested in working with. These data may also be generated from
+scientific workflows or exported from instruments and wearables.
+
+* Examples: [physcraper](https://github.com/pyOpenSci/software-submission/issues/26)
+
+
+### Data deposition
+
+Tools for depositing data into scientific research repositories.
+
+* Examples: [This is an example from rOpenSci - eml](https://github.com/ropensci/software-review/issues/80)
+
+### Data validation and testing:
+
+Tools that enable automated validation and checking of data quality and
+completeness. These tools should be able to support scientific workflows.
+
+* Example: [pandera](https://github.com/pyOpenSci/software-submission/issues/12)
+
+### Scientific software wrappers
+
+Scientific software wrappers refer to packages that provide a Python interface
+for existing scientific packages written in other languages.
+
+These programs must be specific to scientific domains and not general computing
+utilities. Wrappers must provide significant added value to the scientific
+ecosystem be it in data handling, or improved installation processes for
+Python users.
+
+We strongly encourage submissions that wrap tools that are open-source and
+using an OSI-approved license. Exceptions will be evaluated case-by-case,
+considering whether open-source options exist.
+
+<!-- TODO: need an example for this category -->
+* Examples: We don't have a package in this category yet - *Could be your package?*
+
+### Workflow Automation & versioning
```

Comment:
For consistency, "Automation" doesn't need to be capitalized.

Reply:
fixed! thanks!